### ISSUE TITLE
Why not found on NuGet？

### ISSUE THREAD

**Issue by scotly** [wilchn/BlazingQuartz#40]
Is the package name BlazingQuartz?

**Comment by zackasattackas** [comment]
NuGet package for UI components would be awesome!

**Comment by dhadamss** [comment]
Yes, definitely, a NuGet package for the UI components would make converting off Crystal Quartz relatively seamless and increase the adoption of this superior UI.

**Comment by SIGEER2222** [comment]
If it hasn't been updated after a year, does this mean we need to release our own NuGet package?
